src/tools/info/tl_info.c

```diff
@@ -41,14 +41,42 @@
         if (ucs_test_all_flags(_cap_flags, \
                                UCT_IFACE_FLAG_##_name##32 | UCT_IFACE_FLAG_##_name##64)) \
         { \
-            printf("#         %12s: 32, 64 bit%s\n", s, domain); \
+            printf("#         %12s: 32, 64 bit%s (deprecated)\n", s, domain); \
         } else { \
-            printf("#         %12s: %d bit%s\n", s, \
+            printf("#         %12s: %d bit%s (deprecated)\n", s, \
                    ((_cap_flags) & UCT_IFACE_FLAG_##_name##32) ? 32 : 64, domain); \
         } \
         free(s); \
     }
 
+#define _PRINT_ATOMIC_POST(_name, _cap, _op, _suffix) \
+    if (((_cap).atomic32._op & UCS_BIT(UCT_ATOMIC_OP_##_name)) ||  \
+        ((_cap).atomic64._op & UCS_BIT(UCT_ATOMIC_OP_##_name))) {  \
+        char *s = strduplower(#_name); \
+        char amo[256] = "atomic_" _suffix; \
+        char *domain = ""; \
+        strcat(amo, s); \
+        free(s); \
+        if ((_cap).flags & UCT_IFACE_FLAG_ATOMIC_CPU) { \
```

Comment:
the domain can be printed once per iface and not per every atomic type

Reply:
print domain as separate capability line?

Reply:
yes, if at least one atomic flag is present

Reply:
done